src/components/PurchaseOrder/ongoingOgderInfo/OngoingInfoForm.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Col,
+  Row,
+} from '@folio/stripes/components';
+
+import {
+  FieldRenewalInterval,
+  FieldRenewalDate,
+  FieldRenewalPeriod,
+  FieldIsManualRenewal,
+  FieldRenewalSubscription,
+  FieldReviewDate,
+  FieldOngoingInfoNotes,
+} from '../../../common/POFields';
+
+import { isWorkflowStatusIsPending } from '../util';
+
+const OngoingInfoForm = ({
+  order,
+  formValues = {},
```

Comment:
it's required property, do you need default value?

Reply:
please pass required values instead of `formValues` object, looks like you need only `isSubscription`